app/src/features/projects/view/ProjectDetails.tsx

```diff
@@ -10,20 +13,120 @@ export interface IProjectDetailsProps {
  *
  * @return {*}
  */
-const ProjectDetails: React.FC<IProjectDetailsProps> = () => {
+const ProjectDetails: React.FC<IProjectDetailsProps> = (props) => {
+  const projectData = props.projectData;
+
+  // TODO this is using IProject in the mean time, but will eventually need something like IProjectRecord
+  //const [project ] = useState<IProject | null>(null);
```

Comment:
This comment can be removed.  It wont be relevant given the newer structure to these pages